WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -173,35 +196,39 @@ - (void)configureBlogDetailHeader
                                                                  multiplier:1.0
                                                                    constant:headerWidth]];
         // Center the headerView inside the wrapper
-        [headerWrapper addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+        [constraints addObject:[NSLayoutConstraint constraintWithItem:self.headerView
```

Comment:
Of course! Somehow anchors completely slipped my mind – I've updated this to use them.